pwndbg/commands/heap.py

```diff
@@ -95,6 +95,43 @@ def format_bin(bins, verbose=False, offset=None):
     return result
 
 
+parser = argparse.ArgumentParser()
+parser.description = "Template heap command. You can ignore this."
+parser.add_argument("addr", type=int, help="Address of a chunk header.")
+
+
+@pwndbg.commands.ArgparsedCommand(parser)
+@pwndbg.commands.OnlyWhenRunning
+@pwndbg.commands.OnlyWithResolvedHeapSyms
+@pwndbg.commands.OnlyWhenHeapIsInitialized
+def template_heap_command(addr):
```

Comment:
Maybe we should just make this command like `malloc_chunk`?

Reply:
I was hoping `malloc_chunk` would eventually be as simple as:
`print(chunk)` or `chunk.pretty_print()`

For now I'd argue keeping this template as simple as possible may be the best way to show new contributors how to use the various heap classes.